client/flutter/ios/Runner/Info.plist

```diff
@@ -30,6 +30,12 @@
 	<string>$(FLUTTER_BUILD_NUMBER)</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>fetch</string>
```

Comment:
~do we need `fetch` and `processing` if we're not doing anything with the pushes in the background? or does FCM do some of its own stuff behind the scenes when receiving a push?~ **update:** I see that this is in the [`firebase_messaging` docs](https://pub.dev/packages/firebase_messaging#ios-integration)